app/src/main/java/com/example/pocketcloset/Fragments/CameraFragment.java

```diff
@@ -96,8 +116,65 @@ public void onClick(View v) {
                             }
         });
 
+        ibAdd.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                Intent intent = new Intent(getContext(), SaveClothingView.class);
+                ByteArrayOutputStream stream = new ByteArrayOutputStream();
+                bitmap.compress(Bitmap.CompressFormat.PNG, 20, stream);
+                byte[] bitmapBytes = stream.toByteArray();
+                ParseFile image = new ParseFile("clothingPhoto", bitmapBytes);
+                try {
+                    image.save();
+                } catch (ParseException e) {
+                    e.printStackTrace();
```

Comment:
1. Consider presenting an error toast to notify the user of the error.
2. You likely don't want to start the SaveClothingView intent flow below on line 133, but instead go back from this view.

This is a corner case error condition though. I'd recommend just opening an Issue and resolve at the end if there's time.